apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -615,23 +615,16 @@ const FundIssues = ({ issues, mode }) => {
   const bountylessIssues = []
   const alreadyAdded = []
 
-  if (!tokens.length || !tokenDetails.balance) {
+  // FIXME: initialize bounties & tokenDetails better 😝
+  if (!tokens.length || !tokenDetails.balance) return null
+
+  if (fundsAvailable.toString() === '0') {
     return (
-      <div css={`margin-top: ${2 * GU}px`}>
-        <VaultDiv><Icon /></VaultDiv>
-        <Text color={`${theme.surfaceContentSecondary}`} size='large' >
-          <div>
-            <br />
-            Your base rate has not been set and you do not have
-            any tokens in your Vault.
-            <br /> <br />
-            Once you have tokens in your Vault, you will be
-            able to begin funding issues.
-            <br /> <br />
-            <Button wide onClick={closePanel} mode="strong">Cancel</Button>
-          </div>
-        </Text>
-      </div>
```

Comment:
This was an alternative "no funds available" message that never actually showed up, because the logic doesn't do what it looks like it does. `tokens.length` is only `0` when `appState` hasn't yet loaded, and `tokenDetails.balance` will never be `0`, it will be `"0"`. So in fact, this showed up briefly before our `useEffect` hooks get called. As soon as those get called, this empty message would never show up. The time between this first & second render was so small that I doubt anything ever got painted to the DOM.